### PR TITLE
Ycloud debian9 and ubuntu16 docker images

### PR DESCRIPTION
**Pull request by leki75**

### CODE REVIEW

**Comment by tibkiss** [comment]
+1, but I'd be more confident if Shane Kumpf could also take a look.

File: docker/debian9/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,82 @@
+FROM debian:9 as builder
```
Comment (skumpf) [inline docker/debian9/Dockerfile]: Note that multi-stage builds require a recent version of docker. Make sure where you plan to build supports this.

**Comment by skumpf** [comment]
+1 lgtm

File: docker/debian9/Dockerfile (hunk 1 above)
Comment (leki75) [reply docker/debian9/Dockerfile]: Thank you for your comment. At the moment we build docker images locally but contacted with @hortonworks/releng team about upgrading docker on jenkins slaves.

**Pull request merged**